docs/manual/docs/examples/spo/adding-app-catalog.md

```diff
@@ -0,0 +1,18 @@
+# Hide SharePoint list from Site Contents
```

Comment:
Could you also add references to both files in mkdocs.yml so that they get included in the docs when we build them?

Reply:
Done.